DictionaryExamples.cs

```diff
@@ -8,15 +8,29 @@ namespace TranslatorTextQuickStart
 {
     class Program
     {
-        static string host = "https://api.cognitive.microsofttranslator.com";
+        private const string key_var = "TRANSLATOR_TEXT_SUBSCRIPTION_KEY";
+        private static readonly string subscription_key = Environment.GetEnvironmentVariable(key_var);
+
+        private const string endpoint_var = "TRANSLATOR_TEXT_ENDPOINT";
+        private static readonly string endpoint = Environment.GetEnvironmentVariable(endpoint_var);
+
+        static Program()
+        {
+            if (null == subscription_key)
+            {
+                throw new Exception("Please set/export the environment variable: " + key_var);
+            }
+            if (null == endpoint)
+            {
+                throw new Exception("Please set/export the environment variable: " + endpoint_var);
+            }
+        }
+
         static string path = "/dictionary/examples?api-version=3.0";
```

Comment:
```suggestion
        static string route = "/dictionary/examples?api-version=3.0";
```

This matches the other samples.

Reply:
Done.